cranelift-wasm/src/translation_utils.rs

```diff
@@ -65,7 +66,7 @@ pub struct Global {
     pub initializer: GlobalInit,
 }
 
-/// Globals are initialized via the four `const` operators or by referring to another import.
+/// Globals are initialized via the five `const` operators or by referring to another import.
```

Comment:
Maybe we could just get rid of the manual count in the comment, I don't think it gives us much.